File: client-flutter/lib/config/api_config.dart
/// API configuration pointing to Railway NestJS backend
class ApiConfig {
  ApiConfig._();

  /// Railway NestJS backend
  static const String baseUrl =
      'https://admin-backend-php-production.up.railway.app/';

  /// Auth header name (token-based auth)
  static const String authHeader = 'X-Auth-Token';

  /// Request timeout
  static const Duration timeout = Duration(seconds: 15);
}

File: client-ios/SihlClient/Config/APIConfig.swift
import Foundation

/// API-Konfiguration — zeigt auf das Railway NestJS Backend.
/// Pendant zu Flutter `api_config.dart`.
enum APIConfig {
    /// Railway NestJS backend
    static let baseURL = URL(string: "https://admin-backend-php-production.up.railway.app/")!

    /// Auth-Header-Name (token-basierte Auth)
    static let authHeader = "X-Auth-Token"

    /// Request-Timeout
    static let timeout: TimeInterval = 15
}
